tests/neg/namedTypeParams.scala

```diff
@@ -24,4 +24,10 @@ object Test {
 
   f[Y = String][X = Int](1, "") // error: illegal repeated type application
   f[Y = String][Int](1, "") // error: illegal repeated type application
+
+  import scala.language.experimental.clauseInterleaving
```

Comment:
If you want this `import` to apply only locally, it's best to explicitly create a separate scope.